validator_statistics/Cargo.toml

```diff
@@ -11,6 +11,8 @@ anyhow = { workspace = true }
 eth1_api = { workspace = true }
 fork_choice_control = { workspace = true }
 helper_functions = { workspace = true }
+tracing = { workspace = true }
+logging = { workspace = true }
```

Comment:
[**chore**](https://conventionalcomments.org/): Keep dependencies in alphabetical order